doc/ni_by_example_2.md

```diff
@@ -484,31 +542,62 @@ Files are often stored in compressed form on HDFS, so `hdfst` is usually the ope
 
 Also, note that the paths for the HDFS I/O operators must be absolute; thus HDFS I/O operators start with **three** slashes, for example: `$ ni hdfst:///user/bilow/data ...`
 
+
 ####Using HDFS paths in Hadoop Streaming Jobs:
 
-If you want to use data in HDFS for Hadoop Streaming jobs, you need to use a quoted path, as in:
+If you want to use data in HDFS for Hadoop Streaming jobs, you need to use the path as literal text, which uses the `i` operator (explained in more detail below)
 
 ```
-$ ni \'hdfst://<abspath> HS...
+$ ni ihdfst://<abspath> HS...
 ```
 
-If you do not the quote path, as in:
+This will pass the directory path directly to the Hadoop Streaming job. If you do not use path, as in:
 ```
 $ ni hdfst://<path> HS...
 ```
 
-`ni` will read all of the data out of 
+`ni` will read all of the data out of HDFS, stream that data to a new HDFS folder, and then run the Hadoop job using the name of the folder that has The path must be quoted so that `ni` knows to get the data during the Hadoop job, and not collect the data, package it with itself, and then send the packaged data as a `.jar`.
```

Comment:
This is almost true: if you don't quote a path, ni will stream the data into an HDFS temp-path rather than packaging it with the jar.